packages/gatsby/src/utils/webpack-utils.js

```diff
@@ -326,7 +342,28 @@ module.exports = async ({
 
       return {
         test: /\.(js|mjs)$/,
-        exclude: /@babel(?:\/|\\{1,2})runtime|core-js/,
+        exclude: modulePath => {
+          if (vendorRegex.test(modulePath)) {
+            // If dep uses Gatsby, exclude
+            if (
```

Comment:
In this one I would drop the else if and just use a regular if
```
if (
  modulesThatUseGatsby.some(module =>
    modulePath.includes(module.path)
  )
) {
  return true
}
// If dep is babel-runtime or core-js, exclude
if (/@babel(?:\/|\\{1,2})runtime|core-js/.test(modulePath)) {
  return true
}
```

Reply:
Yes! Doing this